docs/platform/contracts/broadcast.md

```diff
@@ -0,0 +1,12 @@
+**Usage**: `sdk.platform.contracts.broadcast(contract, identity)`    
+**Description**: This method will create an `application` or `user` new identity for you. 
+
+Parameters: 
+
+| parameters                | type      | required       | Description                                                       |  
+|---------------------------|-----------|----------------| -----------------------------------------------------------------	|
+| **contract**              | Contract  | yes            | A valid [created contract](/platform/contracts/create.md) |
+| **identity**              | Identity  | yes            | A valid [registered identity](/platform/identities/register.md) |
```

Comment:
Might be important to list application identity here (since user identity type will not be able to register a contract)
```suggestion
| **identity**              | Identity  | yes            | A valid [registered `application` identity](/platform/identities/register.md) |
```